test/function.c

```diff
@@ -12,3 +12,8 @@ int fooing(void);
  * Empty parameters.
  */
 int fooation();
+
+/**
+ * Array parameters.
+ */
+void foorray(const double array[], int x[5]);
```

Comment:
Maybe we can add a couple of more cases for good measure: `[1][2]` and `[1] [2]` (note the space). These should be ok unless clang introduces / keeps spaces between the bracket elements.

Reply:
Thanks, good idea, added and updated the branch.